bitcoinplusplus/layer-2/clara-shikhelman:-unjamming-lightning.md

```diff
@@ -0,0 +1,331 @@
+---
+title: "Clara Shikhelman: Unjamming Lightning"
+transcript_by: NeroCherubino via review.btctranscripts.com
+media: https://www.youtube.com/watch?v=S7ZH4hr2FlA
+tags: ["lightning","attacks"]
+speakers: ["Clara Shikhelman"]
+categories: ["conference"]
+date: 2023-07-01
+---
+
+**Introduction**
+
+I'm Clara, I'm from Chaincode Labs, and I want to talk to you about an ongoing project that tries to solve the jamming problem in the Lightning Network.
+We'll start from what it is and then we're trying to get it.
+
+**Lightning Network Routing**
+
+A very quick reminder to all of you about Lightning Network routing.
+
+Alice wants to route by Bob to Charlie and we have HTCLC for atomic swaps. The way it goes pretty much Alice talks let's say a hundred satoshis between herself and Bob telling him "if you'll get some secret it will be all yours" and then the instructions to Bob that the next hop is Charlie.
```

Comment:
Here, the word "locks" was wrongly transcribed as "talks".
Also, notice that I removed "let's say" from the sentence and the I added an additional word (is) in parenthesis to make the sentence more understandable.
```suggestion
Alice wants to route by Bob to Charlie and we have HTCLs for atomic swaps. The way it goes pretty much Alice locks a hundred satoshis between herself and Bob telling him "if you'll get some secret it will be all yours" and then the instructions to Bob (is) that the next hop is Charlie.
```